code/modules/shuttle/shuttle.dm

```diff
@@ -490,6 +509,10 @@
 /obj/effect/landmark/shuttle_import
 	name = "Shuttle Import"
 
+// Never move the shuttle import landmark, otherwise things get WEIRD
+/obj/effect/landmark/shuttle_import/onShuttleMove()
+	. = FALSE
```

Comment:
return FALSE

Reply:
I've been told that . is faster than an early return.

Reply:
@RemieRichards your ghost will haunt us for all eternity

Reply:
I mean, the use of . over return is still in the CONTRIBUTING.md; is it no longer true?

Reply:
The general maintainer consensus is it's such a small performance difference that `return` is preferred at the end of procs for readability

Reply:
@duncathan to be entirely fair, I haven't requested that one in quite a long time, ever since performance was shown (partly by myself, might I add) to be equivalent in most cases.

Reply:
No one told me

Reply:
Well 
1. you've were gone for some time.
2. the time you've been back it hasn't come up.
3. I quit.

that sum everything up?

Reply:
This is not an aggressive question. If you quit ,why are you still commenting on stuff on Github?

Reply:
because people ping (above) me or link me funny prs.

Reply:
remie is the ghost of coding past